### PR TITLE
[SMALLFIX]Add javadoc for the class of TestResourcePool in ResourcePoolTest.java

### PR DESCRIPTION
**Pull request by ParanoiaW**
add javadoc to ResourcePoolTest#TestResourcePool

### CODE REVIEW

**Comment by ParanoiaW** [comment]
alluxio-bot, check this please

File: core/common/src/test/java/alluxio/resource/ResourcePoolTest.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 import java.util.concurrent.ConcurrentLinkedQueue;
 
 /**
- * Unit test for {@code ResourcePool} class.
+ * Test Unit for {@code ResourcePool} class.
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/ResourcePoolTest.java]: Please change it back to `Unit test`

File: core/common/src/test/java/alluxio/resource/ResourcePoolTest.java [hunk 2]
```diff
@@ -30,13 +30,27 @@
   @Rule
   public ExpectedException mThrown = ExpectedException.none();
 
+  /**
+   * Test unit for {@code ResourcePool} class.
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/ResourcePoolTest.java]: Constructor for ...

**Comment by ParanoiaW** [comment]
Thanks for your comments and I have fixed it

2017-02-22 0:23 GMT+08:00 Yupeng Fu <notifications@github.com>:

**Comment by ParanoiaW** [comment]
alluxio-bot, check this please

File: core/common/src/test/java/alluxio/resource/ResourcePoolTest.java (hunk 2 above)
Comment (ParanoiaW) [inline core/common/src/test/java/alluxio/resource/ResourcePoolTest.java]: Thanks for your comments and I have updated the javadoc according to your comments.

**Pull request merged**